erpnext/accounts/doctype/sales_invoice/sales_invoice.py

```diff
@@ -867,7 +874,7 @@ def make_gle_for_rounding_adjustment(self, gl_entries):
 					"against": self.customer,
 					"credit_in_account_currency": self.rounding_adjustment,
 					"credit": self.base_rounding_adjustment,
-					"cost_center": round_off_cost_center,
+					"cost_center": self.cost_center or round_off_cost_center,
```

Comment:
`round_off_cost_center or self.cost_center`

Reply:
If user opted to tag cost center in Balance Sheet Accounts then round_off amount should also go to Cost Center selected in parent document otherwise trial balance wouldn't match.

Reply:
ok